core/modules/layout/css/layout.flexible.admin.css

```diff
@@ -24,13 +24,13 @@
 }
 .layout--flexible .layout-editor-block-title {
   padding: 0.4em 1em;
-  margin-bottom: 10px;
+  margin:  0 -.9375rem .9375rem;
 }
 .layout--flexible .layout-editor-block-title .text {
-  font-weight: normal;
+  font-weight: bold;
 }
 .layout--flexible .layout-editor-block-content p {
-  color: #575357;
+  color: black;
```

Comment:
In the "Manage blocks" UI this is `#000`. Can we use that here as well?

Reply:
Sure thing!